back-end/src/sciler/handler/handler.go

```diff
@@ -236,13 +237,41 @@ func (handler *Handler) SendStatus(deviceID string) error {
 			},
 		}
 	} else {
-		return fmt.Errorf("error occurred while sending status of %s, since it is not recognised as a device or timer", deviceID)
+		logrus.Errorf("error occurred while sending status of %s, since it is not recognised as a device or timer", deviceID)
+		return
 	}
-
 	jsonMessage, _ := json.Marshal(&message)
 	logrus.Info("sending status data to front-end: " + fmt.Sprint(message.Contents))
 	handler.Communicator.Publish("front-end", string(jsonMessage), 3)
-	return nil
+}
+
+// SendEventStatus sends the status of events to the front-end
+func (handler *Handler) SendEventStatus() {
+	status := handler.getEventStatus()
+	message := Message{
+		DeviceID: "back-end",
+		TimeSent: time.Now().Format("02-01-2006 15:04:05"),
+		Type:     "event status",
+		Contents: status,
+	}
+	jsonMessage, _ := json.Marshal(&message)
+	logrus.Info("sending event status to front-end")
+	handler.Communicator.Publish("front-end", string(jsonMessage), 3)
+}
+
+// returns json list with json objects with keys ["id", "status"]
+// status is json object with key ruleName and value true (if executed == limit) or false
+func (handler *Handler) getEventStatus() []map[string]interface{} {
+	var list []map[string]interface{}
+	for _, rule := range handler.Config.RuleMap {
+		var status = make(map[string]interface{})
+		status["id"] = rule.ID
+		// TODO when is puzzle finished
```

Comment:
Lets make an issue for this so we don't forget

Reply:
yeah it's a good thing to discuss